README.md

```diff
@@ -59,7 +59,7 @@ BNB will be used to:
 
 Many of the below are the same as or similar to go-ethereum.
 
-For prerequisites and detailed build instructions please read the [Installation Instructions](https://geth.ethereum.org/docs/install-and-build/installing-geth).
+For prerequisites and detailed build instructions please read the [Installation Instructions](https://geth.ethereum.org/docs/getting-started/installing-geth).
 
 Building `geth` requires both a Go (version 1.14 or later) and a C compiler. You can install
```

Comment:
`1.14` seems not correct now.

Reply:
update to `1.18` as per [geth docs](https://github.com/ethereum/go-ethereum).